src/Components/Color/Color.jsx

```diff
@@ -23,20 +22,13 @@ export default function Color({ color, onDelete }) {
   function plsCancelEdit(){
     setIsEdit(true);
   }
-  function handleEdit(event){
-    event.preventDefault();
-    const formData = new FormData(event.target);
-    const data = Object.fromEntries(formData);
-    console.log(data)
-    setCurrentColor({id: color.id, role: data.role, hex: data.hex, contrastText: data.contrastText})
-  }
   /*------------------------------------------*/
 
   return (
-    <div className="color-card" style={{ background: currentColor.hex, color: currentColor.contrastText, }} >
-      <h3 className="color-card-highlight">{currentColor.hex}</h3>
-      <h4>{currentColor.role}</h4>
-      <p>contrast: {currentColor.contrastText}</p>
+    <div className="color-card" style={{ background: color.hex, color: color.contrastText, }} >
+      <h3 className="color-card-highlight">{color.hex}</h3>
+      <h4>{color.role}</h4>
+      <p>contrast: {color.contrastText}</p>
       <button type="button" className={isConfirmingDelete ? "" : "hidden"} onClick={plsDelete}>Delete Color</button>
```

Comment:
Please do not use classnames for hiding / showing a button. You should use useStates for this purpose.